content/languages/python/authoring/index.md

```diff
@@ -0,0 +1,214 @@
+---
+kind: tutorial
+languages: [python]
+sidebar: "language:python"
+---
+
+# Python: creating and translating a kata
+
+This article is meant as help for kata authors and translators who would like to create new content in Python. It attempts to explain how to create and organize things in a way conforming to [authoring guidelines](/authoring/guidelines/), what are the most common pitfalls, and how to avoid them.
```

Comment:
```suggestion
This article is meant as help for kata authors and translators who would like to create new content in Python. It attempts to explain how to create and organize things in a way conforming to [authoring guidelines](/authoring/guidelines/), the most common pitfalls and how to avoid them.
```